ods_ci/tasks/Resources/RHODS_OLM/install/oc_install.robot

```diff
@@ -21,6 +27,9 @@ Install RHODS
           FAIL    Provided test envrioment is not supported
       END
   END
+  IF  "${UPDATE_CHANNEL}" != "stable"
```

Comment:
for managed we use beta and self managed we use stable so i think we should change condition